### ISSUE TITLE
Improve home page performance

### ISSUE THREAD

**Issue by Aditeya** [LeWorldAfter/LeWorldAfter.github.io#29]
Use lighthouse in devtools analyze page perfomance.
It'll give tips on what to do.
make sure to check: 
- Performance
- Accessability
- Best practices

The GOAL is to make it load in _**Milliseconds**_ not seconds

**Comment by EltoCode** [comment]
The current source of lag is the wheelnav icons. However, that does not hinder page load times. 

The page takes a long time to load because it needs to pre download the images to perform a smooth background slideshow. I'm afraid nothing can be done about that.

**Comment by Aditeya** [comment]
Yeah but you can convert them to webp and change the logo animation to something faster

**Comment by EltoCode** [comment]
All the bg images are webp already, what do you propose we do for the logo gif?

**Comment by Aditeya** [comment]
The particle.js img files are png. Tou can turn gif into mp4+webm. Itll be significantly smaller

**Comment by EltoCode** [comment]
particle.js files are placeholder, I'll look into webm.
